code/game/objects/random/random.dm

```diff
@@ -998,7 +998,13 @@
 		/obj/item/eightball								= 11,
 		/obj/item/eightball/haunted						= 1,
 		/obj/item/eightball/broken						= 1,
-		/obj/item/weapon/spirit_board					= 5
+		/obj/item/weapon/spirit_board					= 5,
+		/obj/item/device/laser_pointer/red						= 11,
+		/obj/item/device/laser_pointer/green						= 11,
+		/obj/item/device/laser_pointer/blue						= 11,
+		/obj/item/device/laser_pointer/purple						= 11
```

Comment:
Don't you think a total of 44 is an extremely high chance for an item that legitimately can have a combat use (blinding)? Considering most on that list are just toys with next to no use whatsoever, and especially if you compare that to the other 11-chance items. I suggest all of them become become 1, with a weight of 1 in total.